app/views/landings/new_solicitation.html.haml

```diff
@@ -19,6 +19,12 @@
   %section.section.section-grey#section-thankyou
     .container.container-medium
       = render 'new_solicitation_thank_you', solicitation: @solicitation
+      - if landing.logos.present?
```

Comment:
ici j'ai une erreur `undefined local variable or method `landing' for #<#<Class:0x00007f7ee5747758>:0x00007f7ecca024e8>
Did you mean?  @landing`

Reply:
ah bin oui, je suis partie trop vite chez la nounou hier soir